app/controllers/filler/basicPokemonDetail.js

```diff
@@ -4,11 +4,46 @@ const PokemonModel = require(__appbase + 'models/pokemon'),
     async = require('async'),
     cfg = require(__appbase + '../config');
 
+let fs = require('fs'),
+    basicPokemonDetail = require('../../models/BasicPokemonDetail.js'),
+    pokemonListPath = require('../../../resources/json/pokemonlist.json');
+
 module.exports = {
-    fill: function (callback) {
+    fill:
+
+        function loadBasicPokemonDetails() {
+            logger.info('Loading Basic Pokemon Details to MongoDB');
+
+            let data = JSON.stringify(pokemonListPath, function(key,value) {
+                var pokemonID, pokemonName, gender, male_ratio, female_ratio, breedable;
+                if(key != 'name' && typeof value.name !== "undefined") {
+                    pokemonID = key;
+                    pokemonName = value.name;
+                    gender = value.gender.abbreviation;
+                    male_ratio = value.gender.male_ratio;
+                    female_ratio = value.gender.female_ratio;
+                    breedable = value.gender.breedable;
 
+                    basicPokemonDetail.create({pokemonID: pokemonID, name: pokemonName,
```

Comment:
As discussed yesterday, database operations should be under separate file under `store` directory.